_posts/2024-02-14-provisioning-ubuntu-autoinstall.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Provisioning Ubuntu Autoinstall
+date: 2024-02-14 09:41:32
+author: Maximilian Kolb
+tags:
+- foreman
+- provisioning
+- ubuntu
+- autoinstall
+---
+
+# Provisioning Ubuntu Autoinstall
+
+Foreman supports provisioning hosts running Debian and Ubuntu on VMware, Proxmox, cloud providers such as Microsoft Azure, and bare metal hosts.
+With the Katello plug-in, provisioned and registered hosts receive versioned DEB content based on their lifecycle.
+This blog article is a short summary of _what_ happened behind the scenes to provisioning hosts running Ubuntu 20.04.3+, _why_ you need new templates, and _how_ to use the new deployment mechanism.
+
```

Comment:
`<!--more-->` here, so the rest gets hidden on the index page (otherwise it gets waaaaay to long)

Reply:
Added